wire-ios-data-model/Source/ManagedObjectContext/Migration/CoreDataMessagingMigrationVersion.swift

```diff
@@ -67,8 +68,10 @@ enum CoreDataMessagingMigrationVersion: String, CaseIterable {
 
     var nextVersion: Self? {
```

Comment:
I can not mark the older code below, so I add my comment here: it's safe to migrate directly to the version that runs the custom migration action, so we have less migration steps in between (skip `.version2_113`).
```swift
    var nextVersion: Self? {
        switch self {
        case .version2_115:
            return nil
        case .version2_114:
            return .version2_115
        case .version2_111, 
                .version2_112,
                .version2_113:
            return .version2_114 // destination version runs custom migration actions
        case .version2_110:
            return .version2_111 // destination version runs custom migration actions
        case .version2_107,
                .version2_108,
                .version2_109:
            return .version2_110
        case .version2_106:
            return .version2_107 // destination version runs custom migration actions
```

Reply:
Probably I won't change it for version `.version2_110`, because that's anyway out to the public already and there is basically no benefit and it's easier to understand what happened in the past.

Reply:
@markusfassbender not sure it's 100% safe, as it inferred depending on what we have between versions CoreData could miss some details.

Reply:
So should I change or not?

Reply:
@typfel after discussion, I think it's ok to change as @markusfassbender said. But yes **don't change** `.version2_110`